server/api/index.js

```diff
@@ -11,8 +11,9 @@ const shopify = new Shopify({
 
 const callShopify = () => {
 	const currentDate = new Date(); // Return orders only form current day
+	const getHours = currentDate.getHours();
```

Comment:
If I run this code in my browser console I get `21`, which is not a valid date format. 

Please read the Shopify API documentation to make sure you've understood what format it wants the date in (https://help.shopify.com/en/api/reference/orders/order#index-2019-10). There are a few different date formats that are standardised for situations like this where one party wants to share a data with another party. It's important you get your date in the right format!

Reply:
Will do thank u for reviewing!